ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/hcaptcha/v1/hcaptcha/hcaptcha.html

```diff
@@ -0,0 +1,42 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2024 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<sly data-sly-use.renderer="${'hcaptcha.js'}"
+     data-sly-use.clientlib="${'/libs/granite/sightly/templates/clientlib.html'}"
+     data-sly-use.label="${renderer.labelPath}"
+     data-sly-use.shortDescription="${renderer.shortDescriptionPath}"
+     data-sly-use.longDescription="${renderer.longDescriptionPath}"
+     data-sly-use.errorMessage="${renderer.errorMessagePath}"
+     data-sly-use.questionMark="${renderer.questionMarkPath}"></sly>
+
+<div class="cmp-adaptiveform-hcaptcha"
+     data-cmp-is="adaptiveFormHCaptcha"
+     id="${hcaptcha.id}"
+     data-sly-use.formstructparser="com.adobe.cq.forms.core.components.models.form.FormStructureParser"
+     data-sly-use.hcaptcha="com.adobe.cq.forms.core.components.models.form.HCaptcha"
+     data-sly-use.component="com.adobe.cq.wcm.core.components.models.Component"
+     data-cmp-visible="${hcaptcha.visible ? 'true' : 'false'}"
+     data-cmp-enabled="${hcaptcha.enabled ? 'true' : 'false'}"
+     data-cmp-required="${hcaptcha.required ? 'true': 'false'}"
+     data-cmp-readonly="${hcaptcha.readOnly ? 'true' : 'false'}"
+     data-cmp-adaptiveformcontainer-path="${formstructparser.formContainerPath}">
+    <div data-sly-call="${label.label @componentId=component.id, labelValue=hcaptcha.label.value, labelVisible=hcaptcha.label.visible, bemBlock='cmp-adaptiveform-hcaptcha'}" data-sly-unwrap></div>
+
+    <div class="cmp-adaptiveform-hcaptcha__widget"
```

Comment:
Please don't add redundant data, the parent div already has required attribute set, you need to re-use it. Please ensure that HTML is clean and there is no redundant data in HTML